ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -553,6 +504,95 @@ forkBlockProduction maxBlockSizeOverride IS{..} BlockProduction{..} =
 newtype PRNG = PRNG ChaChaDRG
   deriving NoUnexpectedThunks via UseIsNormalForm PRNG
 
+-- | Data about a block's position in the chain
```

Comment:
Mention that it's *about a block to be forged*, not an existing block. Otherwise the `succ` on L525 is a bit confusing.